package.json

```diff
@@ -440,7 +440,6 @@
     "zlib": "^1.0.5"
   },
   "engines": {
-    "node": "10.15.2",
-    "yarn": "^1.10.1"
+    "node": "10.15.2"
```

Comment:
Since we install node in nm-thirdparty, should we remove this install from npm here?

Reply:
@josh-cogdill @john-gress We should revert the changes to this file. `engines` just specifies compatible versions of `node/npm/yarn`, it doesn't install anything. See [here](https://docs.npmjs.com/files/package.json#engines) for the docs on that field!

Reply:
Done